examples/upgradeable-contracts/forward-calls/lib.rs

```diff
@@ -33,6 +33,30 @@ pub mod proxy {
         admin: AccountId,
     }
 
+    /// The trait provides information about the forward pattern used in this contract.
+    #[ink_lang::trait_definition]
+    pub trait Forwarder {
+        /// Returns the `AccountId` of the contract that calls are forward to.
+        #[ink(message)]
+        fn forward_to(&self) -> AccountId;
+
+        /// Returns the `AccountId` of the admin of the current contract.
+        #[ink(message)]
+        fn admin(&self) -> AccountId;
+    }
+
+    impl Forwarder for Proxy {
+        #[ink(message)]
+        fn forward_to(&self) -> AccountId {
+            self.forward_to
+        }
+
+        #[ink(message)]
+        fn admin(&self) -> AccountId {
+            self.admin
+        }
+    }
+
```

Comment:
```suggestion
```

I would remove it, as it doesn't serve a function to the example. I'd like to keep the examples as minimal as possible to focus on illustrating the point (in this case how to build an upgradeable contract).

PR looks good besides that!